packages/app-project/stores/User/UserPersonalization/UserPersonalization.spec.js

```diff
@@ -213,6 +214,10 @@ describe('Stores > UserPersonalization', function () {
       expect(rootStore.user.personalization.totalClassificationCount).to.equal(0)
     })
 
+    it('should not trigger the child Notifications store to request unread notifications or conversations', function () {
+      expect(talkAPI.get).to.have.not.been.called()
```

Comment:
Not a requirement for this PR, but it might be better to mock the Talk API with `nock`, rather than stub the get request.